js2-mode.el

```diff
@@ -6163,8 +6163,6 @@ its relevant fields and puts it into `js2-ti-tokens'."
                              (setf (js2-token-beg token) (- js2-ts-cursor 2))
                              (js2-skip-line)
                              (setf (js2-token-comment-type token) 'line)
-                             ;; include newline so highlighting goes to end of window
```

Comment:
Yes, just re-read your first comment, and I agree with that too. I will add some cases for that.